pkgs/tools/misc/greenwithenvy/default.nix

```
@@ -1,6 +1,8 @@
 { lib
 , stdenv
 , fetchFromGitLab
+, testVersion
```

```suggestion
, testers
```

this was renamed https://github.com/NixOS/nixpkgs/commit/f1c7f19e49190cf58041908f9c194431e2513688